crates/ruff_python_formatter/resources/test/fixtures/import_black_tests.py

```diff
@@ -0,0 +1,106 @@
+from __future__ import annotations
+
+import argparse
+import glob
+import os
+import pathlib
+
+
+def import_fixture(black_input: pathlib.Path, fixture_set: str, fixture_name: str):
+    output_directory = pathlib.Path(os.path.dirname(__file__)).joinpath("black").joinpath(fixture_set)
+
+    output_directory.mkdir(parents=True, exist_ok=True)
+
+    fixture_path = output_directory.joinpath(fixture_name)
+    expect_path = fixture_path.with_suffix(".py.expect")
+
+    with (
+        black_input.open("r", encoding="UTF8") as black_file,
+        fixture_path.open("w", encoding="UTF8") as fixture_file,
+        expect_path.open("w", encoding="UTF8") as expect_file
+    ):
+        lines = black_file.readlines()
```

Comment:
you should be able to just iterate over the file, lines is the default

Reply:
Right, that also removes the need to use enumerate. Nice.